code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -123,6 +123,10 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
 	recipes = GLOB.metal_recipes
 	return ..()
 
+/obj/item/stack/sheet/metal/suicide_act(mob/living/carbon/user)
+	user.visible_message("<span class='suicide'>[user] begins whacking themselves over the head with \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
```

Comment:
make this play an OOF sound

Reply:
like...an actual roblox oof sound?

Reply:
OOF

Reply:
well...alright

Reply:
@MrDoomBringer this is a bad idea walking roblox sounds are from an evil company i would remove it and never listen to that idiot again

Reply:
well...alright

Reply:
wtf @Armhulen